app/src/main/java/com/github/sdpsharelook/storage/IRepository.kt

```diff
@@ -74,4 +83,13 @@ interface IRepository<T> {
      * @param entity Entity Section
      */
     suspend fun insertSection(entity: Section) = Unit
+
+    /**
+     * Create permanent repository entry
+     *
+     * @param name identifier of entity
+     * @param entity Entity Section
+     */
+    suspend fun deleteSection(entity: Section) = Unit
```

Comment:
No need for delete section and delete word inside Irepo, these should only be inside section and word repo. No need for other repos to implement these